bundles/org.eclipse.ui.workbench.texteditor/src/org/eclipse/ui/texteditor/AccessibleToolBar.java

```diff
@@ -0,0 +1,84 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Vector Informatik GmbH and others.
+ *
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Vector Informatik GmbH - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.ui.texteditor;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.graphics.Color;
+import org.eclipse.swt.layout.GridLayout;
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.swt.widgets.ToolBar;
+import org.eclipse.swt.widgets.ToolItem;
+
+import org.eclipse.jface.layout.GridLayoutFactory;
+
+/**
+ * This class wraps the ToolBar to make it possible to use tabulator-keys to
+ * navigate between the buttons of a ToolBar. For this, we simulate a singular
+ * ToolBar by putting each ToolItem into it's own ToolBar and composing them
+ * into a Composite. Since the "Enter" keypress could not previously trigger
+ * activation behavior, we listen for it manually and send according events if
+ * necessary.
+ *
+ * @since 3.17
```

Comment:
Since this class is not part of the API (as it is package protected), a since tag should not be required, should it? So please remove if possible.

Reply:
Yes, I can remove this, we are not adding any new API.

https://github.com/eclipse-platform/eclipse.platform.ui/commit/0676bfe83ef0e491dfbdd69ffcb617f0957b5df0